docs/get-started/features.md

```diff
@@ -124,15 +124,25 @@ See the full list in [github.com/klauspost/cpuid][klauspost-cpuid].
 
 ### Custom
 
-The Custom feature source allows the user to define features based on a mix of
-predefined rules.  A rule is provided input witch affects its process of
-matching for a defined feature. The rules are specified in the
-nfd-worker configuration file. See
-[configuration](deployment-and-usage.md#configuration) for instructions and
-examples how to set-up and manage the worker configuration.
-
-To aid in making Custom Features clearer, we define a general and a per rule
-nomenclature, keeping things as consistent as possible.
+The Custom feature source allows the user to define features based on a mix of predefined rules. A rule is provided
+input witch affects its process of matching for a defined feature. The rules are specified in the nfd-worker
+configuration file. See
+[configuration](deployment-and-usage.md#configuration) for instructions and examples how to set-up and manage the worker
+configuration.
+
+To aid in making Custom Features clearer, we define a general and a per rule nomenclature, keeping things as consistent
+as possible.
+
+#### Additional configuration directory
+
+Additionally to the rules defined in the nfd-worker configuration file, the Custom feature can read more configuration
+files located in the `/etc/kubernetes/node-feature-discovery/custom.d/` directory. This makes more dynamic and flexible
+configuration easier. This directory must be available inside the NFD worker container, so Volumes and VolumeMounts must
+be used for mounting e.g. ConfigMap(s). The
+[nfd-worker deployment template](https://github.com/kubernetes-sigs/node-feature-discovery/blob/{{ site.release
+}}/nfd-worker-daemonset.yaml.template#L204)
+provides an example for providing Custom configuration with an additional ConfigMap, mounted into the `custom.d`
+directory.
```

Comment:
nit: could you wrap the lines to 80 chars (also elsewhere in this file)? Markdown linter (`mdl`) bugs about these. I haven't integrated that to CI because we don't fully pass muster

Reply:
sure, thx, done